.circleci/config.yml

```diff
@@ -222,6 +222,35 @@ jobs:
             RELEASE_CHANNEL: experimental
           command: ./scripts/circleci/run_devtools_e2e_tests.js
 
+  run_devtools_tests_for_versions:
+    docker: *docker
+    environment: *environment
+    parallelism: *TEST_PARALLELISM
+    parameters:
+      version:
+        type: string
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run: yarn workspaces info | head -n -1 > workspace_info.txt
+      - *restore_node_modules
+      - run:
+          name: Install nested packages from Yarn cache
+          command: yarn --frozen-lockfile --cache-folder ~/.cache/yarn
+      - run: 
+          command: ./scripts/circleci/run_devtools_tests_for_react_versions.js << parameters.version >>
```

Comment:
These two commands going to run in sequence, which will slow down the overall build pipeline. Mind running them in parallel instead?

Reply:
Yup will do!